anndata/tests/test_ref_path.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+import numpy as np
+import pandas as pd
+import scipy.sparse as ssp
+
+from anndata import AnnData
+from anndata._core.ref_path import RefPath, split_paths
+
```

Comment:
Could there also be tests for ambiguities throwing errors here?

Reply:
you mean ambiguities in `AnnData.get_vector`

Reply:
Yes